modules/network-observability-flowcollector-api-specifications.adoc

```diff
@@ -981,16 +1155,20 @@ Type::
 | `string`
 | name of the config map or Secret containing certificates
 
+| `namespace`
+| `string`
+| namespace of the config map or secret containing certificates. If omitted, assumes same namespace as where NetObserv is deployed. If the namespace is different, the config map or the secret will be copied so that it can be mounted as required.
```

Comment:
```suggestion
| namespace of the config map or secret containing certificates. If omitted, assumes the same namespace as where NetObserv is deployed. If the namespace is different, the config map or the secret will be copied so that it can be mounted as required.
```